src/main/kotlin/no/nav/arbeid/pam/ontologindexer/OntologiScheduler.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.arbeid.pam.ontologindexer
+
+import no.nav.arbeid.pam.ontologindexer.service.JobbtittelIndexerService
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+import java.io.IOException
+
+
+@Component
+class OntologiScheduler {
+    private val LOGGER = LoggerFactory.getLogger(OntologiScheduler::class.java)
+
+    @Autowired
+    lateinit var jobbtittelIndexerService: JobbtittelIndexerService
+
+    @Scheduled(cron = CRON_INDEXER)
+    fun leggInnStillingstitler() {
+        LOGGER.info("leggInnStillingstitler scheduled job startet cron:$CRON_INDEXER")
+        jobbtittelIndexerService.indekser()
+        LOGGER.info("leggInnStillingstitler scheduled job avsluttet")
+    }
+
+    @Scheduled(cron = CRON_CLEANUP)
+    fun cleanup() {
```

Comment:
Kunne cleanup vært gjort sammen med leggInn? Er det noen grunn til å ha det som to forskjellige schedules?

Reply:
Slår de sammen